scripts/zap/sample.context

```diff
@@ -0,0 +1,91 @@
+        <desc/>
+        <inscope>true</inscope>
+        <incregexes>https://keycloak-zac-dev.dimpact.lifely.nl.*</incregexes>
+        <incregexes>https://zaakafhandelcomponent-zac-dev.dimpact.lifely.nl.*</incregexes>
+        <tech>
+            <include>Db</include>
+            <include>Db.CouchDB</include>
+            <include>Db.Firebird</include>
+            <include>Db.HypersonicSQL</include>
+            <include>Db.IBM DB2</include>
+            <include>Db.MariaDB</include>
+            <include>Db.Microsoft Access</include>
+            <include>Db.Microsoft SQL Server</include>
+            <include>Db.MongoDB</include>
+            <include>Db.MySQL</include>
+            <include>Db.Oracle</include>
+            <include>Db.PostgreSQL</include>
+            <include>Db.SAP MaxDB</include>
+            <include>Db.SQLite</include>
+            <include>Db.Sybase</include>
+            <include>Language</include>
+            <include>Language.ASP</include>
+            <include>Language.C</include>
+            <include>Language.JSP/Servlet</include>
+            <include>Language.Java</include>
+            <include>Language.Java.Spring</include>
+            <include>Language.JavaScript</include>
+            <include>Language.PHP</include>
+            <include>Language.Python</include>
+            <include>Language.Ruby</include>
+            <include>Language.XML</include>
+            <include>OS</include>
+            <include>OS.Linux</include>
+            <include>OS.MacOS</include>
+            <include>OS.Windows</include>
+            <include>SCM</include>
+            <include>SCM.Git</include>
+            <include>SCM.SVN</include>
+            <include>WS</include>
+            <include>WS.Apache</include>
+            <include>WS.IIS</include>
+            <include>WS.Tomcat</include>
+        </tech>
+        <urlparser>
+            <class>org.zaproxy.zap.model.StandardParameterParser</class>
+            <config>{"kvps":"&amp;","kvs":"=","struct":[]}</config>
+        </urlparser>
+        <postparser>
+            <class>org.zaproxy.zap.model.StandardParameterParser</class>
+            <config>{"kvps":"&amp;","kvs":"=","struct":[]}</config>
+        </postparser>
+        <authentication>
+            <type>2</type>
+            <strategy>EACH_RESP</strategy>
+            <pollurl/>
+            <polldata/>
+            <pollheaders/>
+            <pollfreq>60</pollfreq>
+            <pollunits>REQUESTS</pollunits>
+            <form>
+                <loginurl>https://my-keycloak.url/realms/zaakafhandelcomponent</loginurl>
+                <loginbody>username=[%username%]&amp;password=[%password%]&amp;credentialId=</loginbody>
+                <loginpageurl>https://my-keycloak.url/realms/zaakafhandelcomponent</loginpageurl>
+            </form>
+        </authentication>
+        <users>
+            <user>137;true;ZnVuY3Rpb25lZWxiZWhlZXJkZXI=;2;ZnVuY3Rpb25lZWxiZWhlZXJkZXI=~YUhRV3pXR144bUhx~</user>
```

Comment:
What are these strings in here? Hashed passwords? Is this good/intended?

Reply:
Good one Edgar. This is not intended, but I've not got a solution for this as yet...

Reply:
User hashes have been removed and replaced with a placeholder.